src/components/ExploreList.jsx

```diff
@@ -22,7 +22,7 @@ const ExploreList = () => {
 
   const articleList = articles.map((article, i) => {
     return (
-      <Item data-cy="explore-item" data-id={`explore-item-${i + 1}`} key={i + 1} >
+      <Item as='a' href={`http://localhost:3001/articles/${article.id}`} data-cy="explore-item" data-id={`explore-item-${i + 1}`} key={i + 1} >
```

Comment:
use `as='Link'` from react-router-dom instead of an anchor link. Then you get the same functionality without having to type out the whole URL and can instead point to what component that should be rendered